src/modules/dashboard/components/ColonyDecisions/ColonyDecisionsWrapper.tsx

```diff
@@ -5,13 +5,20 @@ import ColonyNewDecision from '~dashboard/ColonyNewDecision';
 
 import { LOCAL_STORAGE_DECISION_KEY } from '~constants';
 import { DecisionDetails } from '~types/index';
+import { Colony } from '~data/index';
 
 import ColonyDecisions from './ColonyDecisions';
 
+interface Props {
+  colony: Colony;
+  filteredDomainId: number;
+  onDomainChange?: (domainId: number) => void;
+}
+
 const ColonyDecisionsWrapper = ({
   colony,
   filteredDomainId,
-  setDomainIdFilter,
+  onDomainChange,
 }: any) => {
```

Comment:
Don't you want to use the newly defined `Props` here?

Reply:
of course, that was the creation's purpose  🙈